### ISSUE TITLE
Привести в соответствие с 1С метод Элементы.

### ISSUE THREAD

**Issue by ret-Phoenix** [ret-Phoenix/oscript-simple-gui#32]
Сейчас работает как подчиненныеЭлементы в случае поиска по имени

**Comment by nixel2007** [comment]
breaking change?

**Comment by ret-Phoenix** [comment]
@nixel2007 Нет, не думаю, сейчас ищет в пределах подчинения по имени, а будет искать во всей форме. Вероятность того, что в одной форме создали элемент с одним и тем же именем почти равна нулю.
